receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -219,35 +234,149 @@ func TestTranslateV2(t *testing.T) {
 			request: writeV2RequestFixture,
 			expectedMetrics: func() pmetric.Metrics {
 				expected := pmetric.NewMetrics()
+				// For job "service-x/test", no explicit otel_scope is provided.
+				// fall back to BuildInfo defaults.
 				rm1 := expected.ResourceMetrics().AppendEmpty()
-				rmAttributes1 := rm1.Resource().Attributes()
-				rmAttributes1.PutStr("service.namespace", "service-x")
-				rmAttributes1.PutStr("service.name", "test")
-				rmAttributes1.PutStr("service.instance.id", "107cn001")
+				parseJobAndInstance(rm1.Resource().Attributes(), "service-x/test", "107cn001")
 				sm1 := rm1.ScopeMetrics().AppendEmpty()
-				sm1Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
-				sm1Attributes.PutStr("d", "e")
-				sm1Attributes.PutStr("foo", "bar")
-				// Since we don't check "scope_name" and "scope_version", we end up with duplicated scope metrics for repeated series.
-				// TODO: Properly handle scope metrics.
-				sm2Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()
-				sm2Attributes.PutStr("d", "e")
-				sm2Attributes.PutStr("foo", "bar")
+				sm1.Scope().SetName(defaultBuildName)
+				sm1.Scope().SetVersion(defaultBuildVersion)
+				// Expect 2 separate gauge metrics, one per timeseries.
+				m1 := sm1.Metrics().AppendEmpty().SetEmptyGauge()
+				dp1 := m1.DataPoints().AppendEmpty()
+				dp1.Attributes().PutStr("d", "e")
+				dp1.Attributes().PutStr("foo", "bar")
```

Comment:
do you have any reason to change `sm1Attributes := sm1.Metrics().AppendEmpty().SetEmptyGauge().DataPoints().AppendEmpty().Attributes()` ->

m1 := sm1.Metrics().AppendEmpty().SetEmptyGauge()
dp1 := m1.DataPoints().AppendEmpty()
dp1.Attributes().PutStr(...)
dp1.Attributes().PutStr(...)

?

Seems that the previous implementation it's ok